src/Ormolu/Printer/Meat/Declaration/Value.hs

```diff
@@ -181,28 +209,101 @@ p_grhs style (GRHS NoExt guards body) =
       breakpoint
       inci p_body
   where
-    p_body = located body p_hsExpr
-p_grhs _ (XGRHS NoExt) = notImplemented "XGRHS"
+    p_body = located body pretty
+p_grhs' _ _ (XGRHS NoExt) = notImplemented "XGRHS"
+
+p_hsCmd :: HsCmd GhcPs -> R ()
+p_hsCmd = \case
+  HsCmdArrApp NoExt body input arrType _ -> do
+    located body p_hsExpr
+    txt $ case arrType of
+      HsFirstOrderApp -> " -<"
+      HsHigherOrderApp -> " -<<"
+    placeHanging (exprPlacement (unLoc input)) $
+      located input p_hsExpr
+  HsCmdArrForm NoExt form Prefix _ cmds -> banana $ sitcc $ do
+    located form p_hsExpr
+    unless (null cmds) $ do
+      breakpoint
+      inci (sequence_ (intersperse breakpoint (located' p_hsCmdTop <$> cmds)))
+  HsCmdArrForm NoExt form Infix _ [left, right] -> do
+    located left p_hsCmdTop
+    space
+    located form p_hsExpr
+    placeHanging (cmdTopPlacement (unLoc right)) $
+      located right p_hsCmdTop
+  HsCmdArrForm NoExt _ Infix _ _ -> notImplemented "HsCmdArrForm"
```

Comment:
`Infix` should never match if there are more or less than two arguments, hence the `notImplemented`.